ios/FluentUI.Tests/DatePickerControllerTests.swift

```diff
@@ -7,10 +7,8 @@ import XCTest
 @testable import FluentUI
 
 class DatePickerControllerTests: XCTestCase {
-    static let testDateInterval: TimeInterval = 1551903381
```

Comment:
I was going to ask why this number was there to begin with, but looks like it was brought from Fabric (change dates back to #1). Your change seems to make more sense to me though 👍